dev/docker-compose/docker-compose.yml

```diff
@@ -10,6 +10,16 @@ services:
     ports:
       - "5432:5432"
 
+  pgadmin:
+    container_name: pgadmin4_container
+    image: dpage/pgadmin4
+    restart: always
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@admin.com
+      PGADMIN_DEFAULT_PASSWORD: root
+    ports:
+      - "5050:80"
+
```

Comment:
we already have a lot of containers running for local development, is this mandatory?

Reply:
done